### ISSUE TITLE
Response $ref type != struct

### ISSUE THREAD

**Issue by llonchj** [Teamwork/kommentaar#54]
/*
GET /v1/test
tests

Response 200: $ref: bool
*/

throws `could not parse response 200 params: GetReference: could not find type "bool" in package ...`

then tried:

type TestResponse bool

/*
GET /v1/test
tests

Response 200: $ref: TestResponse
*/

and throws: `could not parse response 200 params: GetReference: TestResponse is not a struct or interface but a *ast.Ident`

**Comment by Carpetsmoker** [comment]
Yeah, only structs are supported at the moment. It assumed the response is an object `{..}`.

Why? Because that's what all our APIs return :-)
